### ISSUE TITLE
Optimized sparse conv from open3d

### ISSUE THREAD

**Issue by ifeherva** [keyu-tian/SparK#30]
This repo is using masked dense convolutions because it is optimized in torch. However, would [this](http://www.open3d.org/docs/latest/python_api/open3d.ml.torch.nn.SparseConv.html) implementation speed things up or too complicated getting this work here?

**Comment by keyu-tian** [comment]
Thanks for this! This framework looks easy to use but it doesn't support depthwise or grouped convolution, while many CNNs rely on them (resnext, convnext, mobilenet, etc.). We are still using masked conv for now since we found no framework currently has a sufficiently efficient implementation (i.e., faster than masked conv in torch) of depthwise/grouped convolution on GPUs.

**Comment by xubin04** [comment]
How about torch sparse++?
